torch/_dynamo/variables/base.py

```diff
@@ -163,6 +163,12 @@ def as_specialized(self, tx):
         """
         return self
 
+    def associate(self, real_value):
+        self.real_value = real_value
```

Comment:
could we get this lazily through the source rather than adding it as a property?

Reply:
Yes, we absolutely could, a la `get` in guards.